<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuditsCreate</class>
 <widget class="QWidget" name="AuditsCreate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>900</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>JetBrainsMono Nerd Font</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Création d'audits</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #121212;
color: white;
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayoutMenuAuditsCreate">
     <item row="0" column="4">
      <widget class="QPushButton" name="pushButtonDocumentationAuditsCreate">
       <property name="font">
        <font>
         <family>JetBrainsMono Nerd Font</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #00C853; /* Vert */
    color: white;
    font-weight: bold;
    border-radius: 5px;
    padding: 10px;
}
QPushButton:hover {
    background-color: #009624; / survol */
}</string>
       </property>
       <property name="text">
        <string>Documentation</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButtonAccueilAuditsCreate">
       <property name="font">
        <font>
         <family>JetBrainsMono Nerd Font</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #00C853; /* Vert */
    color: white;
    font-weight: bold;
    border-radius: 5px;
    padding: 10px;
}
QPushButton:hover {
    background-color: #009624; / survol */
}</string>
       </property>
       <property name="text">
        <string>Accueil</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="7">
      <widget class="QPushButton" name="pushButtonDeconnexionAuditsCreate">
       <property name="styleSheet">
        <string notr="true">background-color: #121212;
color: white;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../resources/images/deconnexion.png</normaloff>../resources/images/deconnexion.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButtonActualiteAuditsCreate">
       <property name="font">
        <font>
         <family>JetBrainsMono Nerd Font</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #00C853; /* Vert */
    color: white;
    font-weight: bold;
    border-radius: 5px;
    padding: 10px;
}
QPushButton:hover {
    background-color: #009624; / survol */
}</string>
       </property>
       <property name="text">
        <string>Actualité</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelLogo">
       <property name="minimumSize">
        <size>
         <width>65</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>65</width>
         <height>42</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../resources/images/logo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="1">
      <widget class="QLabel" name="labelUrlAuditsCreate">
       <property name="font">
        <font>
         <family>JetBrainsMono Nerd Font</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Créer un audit:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="labelWarningUrlAuditsCreate">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>JetBrainsMono Nerd Font</family>
         <pointsize>13</pointsize>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayoutUrlAuditsCreate">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEditUrlAuditsCreate">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>JetBrainsMono Nerd Font</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
	border: 1px solid #ffffff;
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCreerAuditsCreate">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>JetBrainsMono Nerd Font</family>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #2E2E2E;      
    color: white;
    border-radius: 5px;
    padding: 10px;
}
QPushButton:hover {
    background-color: #4CAF50;      
}
QPushButton:pressed {
    background-color: #388E3C;      
}</string>
         </property>
         <property name="text">
          <string>Créer</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelNomAuditsSelect">
       <property name="font">
        <font>
         <family>JetBrainsMonoNL Nerd Font</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Audits</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
